### PR TITLE
Dactyl Manuform 4x5 with 5 thumb cluster

### PR DESCRIPTION
**Pull request by ssedrick**
Created a new version of the dactyl manuform. The 4x5 uses 6 keys in the thumb cluster, so needed a new one that supports only a 5 key thumb cluster

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [X] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/dactyl_manuform/4x5_5/keymaps/default/config.h [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Copyright 2022 Shem Sedrick (@ssedrick)
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/4x5_5/keymaps/default/config.h:1]: This entire file could be removed, actually.

File: keyboards/handwired/dactyl_manuform/4x5_5/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                                   ____,    ____,             ____,    ____,
+                                   ____,    ____,             ____,    ____
+    )
+};
+
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  // If console is enabled, it will print the matrix position and status of each key pressed
+#ifdef CONSOLE_ENABLE
+    uprintf("KL: kc: 0x%04X, col: %u, row: %u, pressed: %b, time: %u, interrupt: %b, count: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed, record->event.time, record->tap.interrupted, record->tap.count);
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/4x5_5/keymaps/default/keymap.c:137]:
```suggestion
    uprintf("KL: kc: 0x%04X, col: %u, row: %u, pressed: %d, time: %u, interrupt: %d, count: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed, record->event.time, record->tap.interrupted, record->tap.count);
```

File: keyboards/handwired/dactyl_manuform/4x5_5/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+            return false;
+            break;
+    }
+    return true;
+}
+
+void persistent_default_layer_set(uint16_t default_layer) {
+    eeconfig_update_default_layer(default_layer);
+    default_layer_set(default_layer);
+}
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/4x5_5/keymaps/default/keymap.c:159]:
```suggestion
```

File: keyboards/handwired/dactyl_manuform/4x5_5/keymaps/ssedrick/config.h [hunk 4]
```diff
@@ -0,0 +1,14 @@
+// Copyright 2022 Shem Sedrick (@ssedrick)
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/4x5_5/keymaps/ssedrick/config.h:1]: same, can be removed.

File: keyboards/handwired/dactyl_manuform/4x5_5/keymaps/ssedrick/keymap.c [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+            return false;
+            break;
+    }
+    return true;
+}
+
+void persistent_default_layer_set(uint16_t default_layer) {
+    eeconfig_update_default_layer(default_layer);
+    default_layer_set(default_layer);
+}
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/4x5_5/keymaps/ssedrick/keymap.c:159]:
```suggestion
```

File: keyboards/handwired/dactyl_manuform/4x5_5/keymaps/ssedrick/keymap.c [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+                                   ____,    ____,             ____,    ____,
+                                   ____,    ____,             ____,    ____
+    )
+};
+
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  // If console is enabled, it will print the matrix position and status of each key pressed
+#ifdef CONSOLE_ENABLE
+    uprintf("KL: kc: 0x%04X, col: %u, row: %u, pressed: %b, time: %u, interrupt: %b, count: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed, record->event.time, record->tap.interrupted, record->tap.count);
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/4x5_5/keymaps/ssedrick/keymap.c:137]:
```suggestion
    uprintf("KL: kc: 0x%04X, col: %u, row: %u, pressed: %d, time: %u, interrupt: %d, count: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed, record->event.time, record->tap.interrupted, record->tap.count);
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by keyboard-magpie — APPROVED** [review]

**Pull request merged**